<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>860</width>
    <height>740</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPort">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmdPort">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstReceive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>461</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Er</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>St</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HT</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>90</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LT</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HV</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LV</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="CustomToolButton" name="btnStatus0">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtAMove">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>460</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="lstMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>551</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdPosition">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AMove:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>命令:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAMove">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>460</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdTemp">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>溫度:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdVoltage">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>電壓:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdRealPos">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>實際:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>位置:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdCommand">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>520</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCali">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>510</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCali">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>510</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cali.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCali">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoComm">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>460</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>通訊</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoVoltage">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>460</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>電壓</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkStatus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>620</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon16.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>570</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSpeed">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>560</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>500</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>571</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeed">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>570</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAuto">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOffset">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>660</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOffset">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>690</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOffset">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>700</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cmdBaud">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>560</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHomeTimout">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>629</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHomeTimeout">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>619</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>629</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>復歸時限:</string>
   </property>
  </widget>
  <widget class="QSlider" name="hsHomeTimeout">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>630</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblMoveTimeout">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>689</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoveTimeout">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>679</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>689</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>轉動逾時:</string>
   </property>
  </widget>
  <widget class="QSlider" name="hsMoveTimeout">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>690</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdSpeed">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>速度:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnT">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>340</y>
     <width>41</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeed_2">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>571</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblHomeTimout_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>630</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblMoveTimeout_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>690</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRunning">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomToolButton</class>
   <extends>QToolButton</extends>
   <header>customtoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
